test/integration_test.go

```diff
@@ -124,97 +124,12 @@ const (
 	TERRAFORM_STATE                                         = "terraform.tfstate"
```

Comment:
The tests that were removed here were refactored out given how big this file is.